tests/test_common/test_vision/test_model/test_allconv.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+
+import torch
+import torch.nn as nn
+
+from code_soup.common.vision.models.allconvnet import AllConv, AllConvNet

```

Comment:
This is inconsistent with our format, we do from code_soup.common.vision.models import AllConv, AllConvNet for which the classes should be a function from the package (aka it should be imported from models/__init__.py